foundation-common/src/main/java/io/servicecomb/foundation/common/utils/FortifyUtils.java

```diff
@@ -16,43 +16,14 @@
 
 package io.servicecomb.foundation.common.utils;
 
-import java.io.File;
```

Comment:
we'd better to remove this class in the future, some of methods in this class, like xml processing can also be deleted.